From acc2913ec106e474a343b548f3f9664aaa2f1918 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lawrence=20Vel=C3=A1zquez?= <larryv@macports.org>
Date: Tue, 27 Jan 2015 21:13:08 -0500
Subject: [PATCH 11/16] (MP) libav: Respect our compiler selection on Darwin
 i386

The override doesn't seem necessary for the configurations we support.
---
 contrib/ffmpeg/module.defs | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/contrib/ffmpeg/module.defs b/contrib/ffmpeg/module.defs
index 9956083..e64b719 100644
--- a/contrib/ffmpeg/module.defs
+++ b/contrib/ffmpeg/module.defs
@@ -109,11 +109,6 @@ FFMPEG.BUILD.extra = V=1
 ## on darwin i386 - unclear if it is a source code or llvm bug.
 ##
 ifeq (darwin-i386,$(BUILD.system)-$(BUILD.machine))
-    ## optimize mode hack: use clang
-    ifneq (none,$(FFMPEG.GCC.O))
-        FFMPEG.GCC.gcc = clang
-    endif
-
     ## debug mode hack: omit frame pointer
     ifneq (none,$(FFMPEG.GCC.g))
         FFMPEG.GCC.args.extra += -fomit-frame-pointer
-- 
2.3.0

